data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,127 @@ mission "Strider notification"
 				decline
+
+
+
+mission "Hai Film Crew [1]"
+	name "Hai Film Crew"
+	description "Bring a movie crew from <origin> to <destination>, so that they can film a horror movie there."
+	minor
+	source
+		attributes hai urban
+		not attributes "uninhabited" "station"
+	destination "Darkrest"
+	passengers 80
+	cargo "film equipment" 50
+	to offer
+		random < 30
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`A rapid tapping on your shoulder dissappiates any intentions you might have had about touring this spaceport in peace. You turn around to find a Hai, short even by their standards, looking up at you.`
+			`	"You're a ship captain, yes? Would you like to be part of cinema history?"`
+			choice
+				`	"I'm not much of a movie buff. I'll pass."`
+					decline
+				`	"Sure, what do you need me to do?"`
+				`	"What do you mean?"`
+			`	"My name is Likeet, and I'm in charge of coordinating transportation for an independent film crew. We need you to bring <bunks> crew members and <tons> of film equipment to Darkrest, then bring us back after about a month. We can pay you <payment>, but the rewards of being involved in an ambitious project such as this will be priceless."`
+			choice
+				`	"Why would you want to go to <planet>? Isn't that an abandoned station?"`
+					goto ask
+				`	"Sure, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label ask
+			`	"Ah, you must not know about <planet>. There are many tales which swirl around the station, chief among them being the existence of ghosts and monsters haunting its corridors. We'd like to take advantage of the popular image of Darkrest in the creation of our horror film by using it as a backdrop. It's sort of the human equivalent of setting your film in a 'haunted' forest. Don't worry, we'll bring enough food to last us for the month."`
+			choice
+				`	"Why do people think it's haunted?"`
+					goto haunted
+				`	"Alright then, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label haunted
+			`	"Back when the station was still in use, there was a mysterious malfunction in one of the hatches that released oxygen out the airlock, suffocating hundreds on board to death. To this day, no one knows what caused it, and legends have grown out of the event."`
```

Comment:
You could probably split this first sentence:
```suggestion
			`	"Back when the station was still in use, there was a mysterious malfunction in one of the hatches which released oxygen out the airlock. Hundreds on board suffocated to death. To this day, no one knows what caused it, and legends have grown out of the event."`
```
Also, the word "that" is vague between whether that hatch was supposed to release oxygen out the airlock or not. You can pick it up from context, but "which" might work better.